build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/CreateAndroidEmulator.cs

```diff
@@ -64,7 +68,14 @@ public override bool Execute ()
 				WriteConfigFile (configPath);
 				return !Log.HasLoggedErrors;
 			}
-
+			Log.LogMessage (MessageImportance.Low, $"Task {nameof (CreateAndroidEmulator)}");
```

Comment:
I thought these "log blocks" were for `xbuild` use, which we haven't supported in ages, and `msbuild` use already provides this information in diagnostic logs.  Do we actually need lines 71-78?